bindings/python/test/trajectory/test_state.py

```diff
@@ -9,11 +9,36 @@
 
 import pytest
 
+import ostk.physics as physics
 import ostk.astrodynamics as astrodynamics
 
 ################################################################################################################################################################
+Instant = physics.time.Instant
+DateTime = physics.time.DateTime
+Time = physics.units.Time
+Scale = physics.time.Scale
+Position = physics.coordinate.Position
+Velocity = physics.coordinate.Velocity
+Frame = physics.coordinate.Frame
 
+State = astrodynamics.trajectory.State
 
+################################################################################################################################################################
+
```

Comment:
add class `TestState` with test functions in there?

Reply:
done